test/api/subgroup/create.test.js

```diff
@@ -34,8 +34,23 @@ test('tribes.subtribe.create', t => {
         t.equal(child, subgroupId, 'link/group-subgroup child')
         t.deepEqual(recps, [groupId], 'link/group-subgroup recps')
 
-        server.close()
-        t.end()
+        server.tribes.get(subgroupId, (err, group) => {
+          t.error(err, 'get subgroup')
+
+          t.deepEqual(
+            group,
+            {
```

Comment:
I think this should also return groupId... you technically have it already as you made the query with it but I've so often just wanted it in the object already ...

Reply:
Yeah :)